models/zod/genre.schema.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+
+//  Zod Schemas
+export const genresResponseSchema = z.array(z.string().min(1));
+
+// Inferred Types
+export type Genre = string;
```

Comment:
it can be nested from GenresResponse
```suggestion
export type Genre = GenresResponse[number];
```

Reply:
It was unused type. It was removed but good point.

Reply:
Seems like resolved